### ISSUE TITLE
Save figures as HTML using PlotLy

### ISSUE THREAD

**Issue by farhi** [McStasMcXtrace/iFit#75]
can be saved locally:

see https://plot.ly/matlab/offline/

then can export mostly everything, locally. A nice replacement for static PNG.
see e.g. https://plot.ly/matlab/3d-surface-plots/

**Comment by willend** [comment]
Looks like a nice option indeed! Does a plot from current iFit repo work with fig2plotly()?

I tried with 1.9beta2 recently, there I had to do e.g.
<pre>
pcolor(m(3).Signal) 
fig2plotly()
</pre>

Otherwise I got:
<pre>
Warning: Error occurred while executing the listener callback for the plotlyfig class
PlotOptions property PostSet event:
Error using matlab.ui.Figure/set
While setting property 'Name' of class 'Figure':
Value must be a string.

Error in plotlyfig/updatePlotOptions (line 630)
            set(obj.State.Figure.Handle, 'Name', obj.PlotOptions.FileName, 'Visible',
            obj.PlotOptions.Visible);

Error in plotlyfig>@(src,event)updatePlotOptions(obj,src,event)

Error in handleFileName (line 9)
            obj.PlotOptions.FileName = parseString(str,interp);

Error in plotlyfig/plotly (line 415)
            handleFileName(obj);

Error in fig2plotly (line 49)
p.plotly;  
> In handleFileName (line 9)
  In plotlyfig/plotly (line 415)
  In fig2plotly (line 49) 
Error using response_handler (line 31)
Uh oh, an error occurred on the server.

Error in makecall (line 32)
    response_handler(resp);

Error in plotly (line 28)
response = makecall(args, origin, structargs);

Error in plotlyfig/plotly (line 432)
                response = plotly(obj.data, args);

Error in fig2plotly (line 49)
p.plotly;
</pre>
